assets/example-form.js

```diff
@@ -5,7 +5,7 @@ validator.addValidator("email", [
     method: function (field) {
```

Comment:
This is utterly misleading. This file has not been renamed; it's been deleted.

`docs/assets/example-form.js` is _generated_ from `assets/example-form.js`, using `npm run build`.